bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest, Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+EMOJI_1 = random.choice(HEART_EMOJIS)
+EMOJI_2 = random.choice(HEART_EMOJIS)
+USER_LOVEFEST = []
+JSON_FILE = open(Path('bot', 'resources', 'VALENTINES', 'bemyvalentine_valentines.json'), 'r', encoding="utf8")
+VALENTINES = load(JSON_FILE)
+
+
+class BeMyValentine:
+    """
+    A cog that sends VALENTINES to other users !
+    """
+    id = Lovefest.role_id
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="lovefest")
+    async def add_role(self, ctx):
```

Comment:
This should be a command group with commands to both add and remove the role. Something like `.valentine subscribe` and `.valentine unsubscribe` or similar.